SWLOR.Game.Server/Service/BaseService.cs

```diff
@@ -1324,7 +1324,7 @@ public static int CalculateMaxFuel(Guid pcBaseID)
 
```

Comment:
I think you removed the 0.02f change, but just calling it out real quick to confirm.

Reply:
Affirmative.